static/js/game.js

```diff
@@ -9,6 +6,8 @@ var KEYCODE_LEFT = 37;
 var KEYCODE_RIGHT = 39;
 var KEYCODE_SPACE = 32;
 
+var shown_hero_properties = {speed:'', life_count:''};
```

Comment:
on page it will be string. Doesn't matter what type here i use this like list of properties and all properties will be objects